transbank/webpay/webpayplus/transaction_commit_response.py

```diff
@@ -0,0 +1,129 @@
+class TransactionCommitResponse:
+    def __init__(self, json_data):
+        self._vci = json_data.get('vci', None)
+        self._amount = json_data.get('amount', None)
+        self._status = json_data.get('status', None)
+        self._buy_order = json_data.get('buy_order', None)
+        self._session_id = json_data.get('session_id', None)
+        self._card_number = json_data.get('card_details').get('card_number', None) \
+            if json_data.get('card_details', None) is not None else None
+        self.accounting_date = json_data.get('accounting_date', None)
+        self.transaction_date = json_data.get('transaction_date', None)
+        self._authorization_code = json_data.get('authorization_code', None)
+        self._payment_type_code = json_data.get('payment_type_code', None)
+        self._response_code = json_data.get('response_code', None)
+        self._installments_amount = json_data.get('installments_amount', None)
+        self._installments_number = json_data.get('installments_number', None)
+        self._balance = json_data.get('balance', None)
+
+    @property
+    def vci(self):
+        return self._vci
+
+    @vci.setter
```

Comment:
Why all those setters? Aren't all the local fields being set on the constructor anyways?

Reply:
Yes! But I do it in case of update an attribute and it is because habits too.

Reply:
But it makes the reader think that those values can be set from elsewhere. If no other piece of code should change those values, you make your code simpler and easier to reason about by not providing setters.